templates/RELEASE_NOTES_unified.htm.tmpl

```diff
@@ -0,0 +1,61 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
why is this file called RELEASE_NOTES_unified? isn't this just for the docs release?

Reply:
In essence this file is a template of "unified release notes of all repos in html format". What the end-purpose of that output is (in this case "docs release") is irrelevant at this level of the logic.